### PR TITLE
Mult and div

### PR DESCRIPTION
**Pull request by lanarimarco**
MULT and DIV implementation

### CODE REVIEW

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt [hunk 1]
```diff
@@ -248,6 +248,40 @@ data class ZAddStmt(
     }
 }
 
+data class MultStmt(
+    val target: AssignableExpression,
+    @Derived val dataDefinition: InStatementDataDefinition? = null,
```
Comment (ftomassetti) [inline rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt]: Can this statement define a new variable?

File: mutes_for_ci/MUTE13_08.rpgle [hunk 2]
```diff
@@ -0,0 +1,92 @@
+   COP* *NOUI
```
[the remark this answers is not in the document]
Comment (lanarimarco) [reply mutes_for_ci/MUTE13_08.rpgle]: Deleted from mute_tests

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt [hunk 3]
```diff
@@ -851,6 +852,44 @@ class InternalInterpreter(val systemInterface: SystemInterface) {
         }
     }
 
+    private fun mult(statement: MultStmt): Value {
+        // TODO When will pass my PR for more robustness replace Value.render with NumericValue.bigDecimal
```
[the remark this answers is not in the document]
Comment (lanarimarco) [reply rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt]: An issue for "nothing more than  a personal reminder", I believed was a too big stuff, but my experience in git is poor, therefore in the future I will treasure your advices.

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt (hunk 1 above)
Comment (lanarimarco) [reply rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt]: Ops!!! Copy and past issue, but... What is the goal of StatementThatCanDefineData?

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt [hunk 4]
```diff
@@ -248,6 +248,40 @@ data class ZAddStmt(
...
+        if (dataDefinition != null) {
+            return listOf(dataDefinition)
+        }
+        return emptyList()
+    }
+}
+
+data class DivStmt(
+    val target: AssignableExpression,
+    @Derived val dataDefinition: InStatementDataDefinition? = null,
```
[the remark this answers is not in the document]
Comment (lanarimarco) [reply rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt]: As above...

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt (hunk 1 above)
Comment (ftomassetti) [reply rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt]: Operations like clear can implicitly define variables

**Pull request merged**